### PR TITLE
Fix #2275: Update versions and change log

### LINKED ISSUE #2275: Update versions and changelog
**Describe the issue**

As mentioned in https://github.com/WebAudio/web-audio-api/issues/2243#issuecomment-726684632, we're missing the two latest published drafts in the editor's draft.  We also need to update the change log to include a section for changes since 2020 (the latest published draft).

### PR DESCRIPTION
**Pull request by rtoy**
We were missing the two latest CR publications so add them.

Also update the change log to include the latest changes since the last published CR.

***
<a href="https://pr-preview.s3.amazonaws.com/rtoy/web-audio-api/pull/2276.html" title="Last updated on Nov 13, 2020, 7:18 PM UTC (bb6cdad)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/2276/d50c68f...rtoy:bb6cdad.html" title="Last updated on Nov 13, 2020, 7:18 PM UTC (bb6cdad)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -12527,6 +12529,30 @@ class Vec3 {
 
 <h2 id="changes">
 Change Log</h2>
+<h3 id="changes-2020-06-11">
```
Comment (rtoy) [inline index.bs:12532]: Previous sections had id's like "changestart<n>", with the most recent being changestart1. I didn't want to renumber everything, so I made this id just include the date of the last revision.

Let me know if I should use the same convention and renumber them.

**Comment by rtoy** [comment]
I just went down the list of closed PRs and added the corresponding issues (if any) to the change log.

File: index.bs (hunk 1 above)
Comment (svgeesus) [reply index.bs:12532]: I use the same convention of an id with a date, for change lists in specs I'm editing

**Review by svgeesus — APPROVED** [review]
Thanks, LGTM

**Pull request merged**